StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -234,6 +234,7 @@ import Foundation
     // MARK: - Unexpected errors
     // These errors should _never happen_ and indicate a problem with the SDK or the Stripe backend.
     case unexpectedPaymentSheetFormFactoryError = "unexpected_error.paymentsheet.formfactory"
+    case unexpectedAddPaymentMethodViewControllerError = "unexpected_error.paymentsheet.addpaymentmethodviewcontroller"
```

Comment:
I wonder if we should coalesce all these to a single "unexpectedPaymentSheetError" or something for these, since the error names end up containing the same context?  e.g. right now we're sending:
- event: "unexpected_error.paymentsheet.addpaymentmethodviewcontroller"
- error_type: "AddPaymentMethodViewController. AddPaymentMethodViewControllerError"

Reply:
Sounds good. Lets introduce a more generic form for paymentsheet views/viewcontrollers
```
    case unexpectedPaymentSheetError = "unexpected_error.paymentsheet"
```